frontend/lib/util/version-info.cpp

```diff
@@ -51,21 +51,19 @@ std::string getMajorMinorVersion() {
   return version;
 }
 
-std::string getVersion(bool developer) {
+std::string getVersion() {
   std::string ret;
   ret = std::to_string(getMajorVersion()) + "." +
         std::to_string(getMinorVersion()) + "." +
         std::to_string(getUpdateVersion());
   if (!CHPL_OFFICIAL_RELEASE) {
-    ret += " pre-release (" + std::string(getBuildVersion(developer)) + ")";
+    ret += " pre-release (" + std::string(gitSHA()) + ")";
   } else {
     // It's is an official release.
-    // Try to decide whether or not to include the BUILD_VERSION
-    // based on its string length. A short git sha is 10 characters.
-    std::string build = std::string(getBuildVersion(developer));
-    if (build.length() > 2 && !developer) {
-      // assume it is a sha, so don't include it
-    } else if (build.compare("0") == 0) {
+    // based on discussion in https://github.com/Cray/chapel-private/issues/4285
```

Comment:
Let's leave issue link to git history / the PR message. If you can offer a short rationale that would fit an a comment, use that; otherwise, I'd suggest simply leaving out this line